cmake/h5z_zfp-config.cmake.in

```diff
@@ -0,0 +1,59 @@
+# h5z_zfp-config.cmake
```

Comment:
How do you like having both the underscore and the dash in the filename? I kinda like `h5z-zfp-config.cmake` a bit better. What do you think?

I know I use underscores elsewhere as in `H5Zzfp_plugin.h` but am wondering if that is best.

Reply:
Personally, I try to avoid the dash in filename. Especially one a Linux instead since it can be interpreted as something else. This is of course my own personal preference which I do not want to enforce on someone. What I propose we could do is first you read `cmake.rst` file and let me know what think about it. Next we could make a decision if we want to change h5z_zfp to h5z-zfp. This change will, for example, mean that the package name needs to be changed from H5Z_ZFP to H5Z-ZFP. I am not sure if CMake will like that.

Reply:
A moment ago, I tried to switch from the name `h5z_zfp` to `h5z-zfp`. It appears CMake has no problem with it, but unfortunately Linux does. It becomes difficult to export the environment variable `H5Z-ZFP_ROOT` or ` H5Z-ZFP_DIR`.  My bash complains about it is not a valid identifier. So I would suggest to to keep the name `h5z_zfp`.